code/modules/food_and_drinks/food.dm

```diff
@@ -1,19 +1,67 @@
 ////////////////////////////////////////////////////////////////////////////////
 /// Food.
 ////////////////////////////////////////////////////////////////////////////////
+GLOBAL_LIST_INIT(temp_stabilizers, typecacheof(list(
+	/obj/structure/closet/crate/secure/hydroponics,
+	/obj/structure/closet/crate/freezer,
+	/obj/structure/closet/crate/hydroponics,
+	/obj/structure/closet/secure_closet/freezer
+	)))
+
 /obj/item/reagent_containers/food
 	possible_transfer_amounts = list()
 	volume = 50	//Sets the default container amount for all food items.
 	container_type = INJECTABLE_1
 	resistance_flags = FLAMMABLE
 	var/foodtype = NONE
 	var/last_check_time
+	var/cares_about_temperature = FALSE
+	var/starting_temp = 200 // Farenheight
```

Comment:
why would I store time as the total milliseconds since the unix epoch? dumb idea